install_config/install/advanced_install.adoc

```diff
@@ -617,6 +620,26 @@ value; you only need to set this if you want your `git clone` operations to use
 a different value.
 |===
 
+[[advanced-install-configuring-firewalls]]
+=== Configuring the Firewall
+
+{product-title} uses iptables as the default firewall, but you can switch to use firewalld in the install.
+
+Because iptables is the default filewall, {product-title} is designed to have it
```

Comment:
@knobunc This goes against what @sdodson says above: "the unreleased versions of the 3.5 installer defaulted to firewalld but we reverted that change just before release. So at no time has firewalld been the default in a released version."

Am I misunderstanding this? iptables is still the default, right?